apps/myaccount/src/components/linked-accounts/linked-accounts.tsx

```diff
@@ -108,27 +108,12 @@ export const LinkedAccounts: FunctionComponent<LinkedAccountsProps> = (props: Li
                 dispatch(getProfileLinkedAccounts());
             })
             .catch((error) => {
-                if (error.response && error.response.data && error.response.data.detail) {
-                    onAlertFired({
-                        description: t(
-                            "userPortal:components.linkedAccounts.notifications.addAssociation.error.description",
-                            { description: error.response.data.detail }
-                        ),
-                        level: AlertLevels.ERROR,
-                        message: t(
-                            "userPortal:components.linkedAccounts.notifications.addAssociation.error.message"
-                        )
-                    });
-
-                    return;
-                }
-
                 onAlertFired({
-                    description: t(
+                    description: error?.response?.data?.description ?? t(
                         "userPortal:components.linkedAccounts.notifications.addAssociation.genericError.description"
-                    ),
+                    ) ,
                     level: AlertLevels.ERROR,
-                    message: t(
+                    message: error?.response?.data?.message ?? t(
```

Comment:
Is this the backend error we are showing here ? If shall we map it to a front-end error

Reply:
Resolved by 97406f11